static/css/devhub/base.less

```diff
@@ -100,16 +164,17 @@ section {
       margin-left: 10px;
     }
   }
-}
 
-.Logo--secondary::before {
-  background: url('../../img/developers/addons-logo-black.svg') no-repeat left center;
-  background-size: contain;
+  .scheme-medium & {
+    background: url('../../img/developers/addons-logo-black.svg') no-repeat left center;
+    background-size: contain;
+  }
 }
 
 .DevHub-Navigation {
-  background-color: @primary-section-background;
-  padding: 0 20px 20px;
+  > div {
```

Comment:
Yes, I'm relatively sure I need it (checking again) to make it easier to centralize content in a component (and make it centered based on `max-width` and not just center all separate elements) . I think that was why I introduced the separate div.